assets/src/dashboard/karma/fixture.js

```diff
@@ -196,9 +196,13 @@ export default class Fixture {
     this._container = container;
     this._screen = screen;
 
-    // @todo: find a stable way to wait for the story to fully render. Can be
-    // implemented via `waitFor`.
-    return Promise.resolve();
+    // Check to see if Google Sans font is loaded.
+    await waitFor(async () => {
+      await document.fonts.load('0 Google Sans', '');
+      if (!document.fonts.check(' 0 Google Sans', '')) {
```

Comment:
Same question about syntax here. Just want to confirm that it's correct.

Also, there's an errant leading space.